tests/plugins/slow_start.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+"""This plugin is used to check that updated connection hints work properly.
+
+"""
+from pyln.client import Plugin
+
+import socket
+import time
+
+plugin = Plugin()
+
+
+@plugin.init()
+def init(configuration, options, plugin):
+    plugin.keep_alive = True
+
+
+@plugin.async_method('waitconn')
+def wait_connection(request, plugin):
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    addr = ('localhost', 6669)
+    sock.bind(addr)
+    sock.listen(1)
+    print("listening for connections")
+
+    # We are a one and done socket connection!
+    conn, client_addr = sock.accept()
+    try:
+        print("connection from {}".format(client_addr))
+        data = conn.recv(32)
+        print(data)
```

Comment:
Won't this print `repr(data)` which is kind of hard to verify? Should be hex-encoded.

Reply:
it will. This isn't really needed for the test, I just liked seeing it spew garbage. Removed.